qutip/bloch.py

```diff
@@ -353,23 +366,27 @@ def add_states(self, state, kind='vector'):
                    expect(sigmaz(), st)]
 
             if kind == 'vector':
-                self.add_vectors(vec)
+                self.add_vectors(vec, alpha=alpha)
             elif kind == 'point':
-                self.add_points(vec)
+                self.add_points(vec, alpha=alpha)
 
-    def add_vectors(self, vectors):
+    def add_vectors(self, vectors, alpha=1.0):
         """Add a list of vectors to Bloch sphere.
 
         Parameters
         ----------
         vectors : array_like
             Array with vectors of unit length or smaller.
+
+        alpha : float, default=1.
+            Transparency value for the vectors. Values between 0 and 1.
         """
         if isinstance(vectors[0], (list, tuple, ndarray)):
             for vec in vectors:
                 self.vectors.append(vec)
         else:
             self.vectors.append(vectors)
+        self.vector_alpha.append(alpha)
```

Comment:
When vectors is a list of vectors, we should add an alpha value for every vector in the list -- otherwise the wrong alpha values will be applied if one does, e.g.:
```python
b.add_vectors([psi0, psi1], alpha=0.5)
b.add_vectors([psi2, psi3], alpha=1.0)
```
Currently `psi0` and `psi2` will have alpha 0.5 and `psi1` and `psi3` will have alpha 1.0, which is not correct.

Reply:
Implemented.